terraform/events.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_iam_role" "cloudwatch-batch-trigger-role" {
+  name = "cloudwatch-batch-trigger-role"
+  assume_role_policy = templatefile("${path.module}/iam_templates/trust_policy.json", {
+    trust_services = ["events"]
+  })
+}
+
+
+resource "aws_iam_role_policy_attachment" "cloudwatch-can-trigger-batch-jobs" {
+  role       = aws_iam_role.cloudwatch-batch-trigger-role.name
+  policy_arn = aws_iam_policy.can-submit-aspen-batch-job.arn
+}
+
+
+resource "aws_cloudwatch_event_rule" "ingest-gisaid-rule" {
+  name                = "ingest-gisaid"
+  description         = "Ingest GISAID data daily"
+  schedule_expression = "cron(0 0 ? * 1-5 *)"
+}
+
+
+resource "aws_cloudwatch_event_target" "ingest-gisaid-target" {
+  target_id = "ingest-gisaid-target"
+  rule      = aws_cloudwatch_event_rule.ingest-gisaid-rule.name
+  arn       = aws_batch_job_queue.aspen-batch-job-queue.arn
+  role_arn  = aws_iam_role.cloudwatch-batch-trigger-role.arn
+
+  batch_target {
+    job_name       = "ingest-gisaid"
+    job_definition = aws_batch_job_definition.aspen-batch-job-definition.name
+  }
+
+  input = <<-CONTAINER_OVERRIDES
```

Comment:
can you add a comment for what this is doing? I'm not familiar with `CONTAINER_OVERRIDES`

Reply:
done.